### DISCUSSION TITLE
How to draw a cup pattern in lightweight-charts?

### DISCUSSION THREAD

**Discussion by mirzaeianfar** [tradingview/lightweight-charts#910] [Q&A]
![Et09upicture129](https://user-images.githubusercontent.com/53333713/142736906-64249178-cb38-406b-90c7-74fcf31f9779.jpg)

**Comment by timocov** [comment]
See https://github.com/tradingview/lightweight-charts/issues/906#issuecomment-975383224

**Comment by mariamlouis** [comment]
I am just beginner to use trading view
Please help how to include in a project using visual studio and C#
